### PR TITLE
Cpdrp 815 content

### PR DESCRIPTION
**Pull request by Lloyd-K**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDRP-815?atlOrigin=eyJpIjoiYjNkYmQ5ZjM1MWZmNDAzODk5ZDRmYTc5ZjhlZmI4MmYiLCJwIjoiaiJ9

### CODE REVIEW

File: app/views/check_account/show.html.erb [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-    <ul>
-      <li><%= govuk_link_to "sign in to DfE", "https://services.signin.education.gov.uk/" %> and check that your GIAS email is up to date.</li>
-      <li><%= govuk_link_to "send a new link", resend_email_request_nomination_invite_path %></li>
-      </ul>
-    <h2 class="govuk-heading-m">New teachers or mentors</h2>
-    <p class="govuk-body">You only need to sign in if you’re using our accredited training materials. We will send you sign in details before you start your programme.</p>
+    <p class="govuk-body">You can <%= govuk_link_to "send your school a link", resend_email_request_nomination_invite_path %> to use this service.</p>
+    
+    <h2 class="govuk-heading-m">School induction tutors</h2>
+    <p class="govuk-body">If you've been asked to manage your early career teacher training programme, check your school has used our service to tell us your name and email address.</p>
```
Comment (tonyheadford) [inline app/views/check_account/show.html.erb:11]:
```suggestion
    <p class="govuk-body">If you’ve been asked to manage your early career teacher training programme, check your school has used our service to tell us your name and email address.</p>
```

File: app/views/check_account/show.html.erb [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
+      
+    <h2 class="govuk-heading-m">Early career teachers and mentors</h2>
+    <p class="govuk-body">We'll email you directly if you need to use this service. Contact your school for more information.</p>
```
Comment (tonyheadford) [inline app/views/check_account/show.html.erb:14]:
```suggestion
    <p class="govuk-body">We’ll email you directly if you need to use this service. Contact your school for more information.</p>
```

File: app/views/nominations/request_nomination_invite/choose_location.html.erb [hunk 3]
```diff
@@ -11,7 +11,7 @@
               @local_authorities,
               :id,
               :name,
-              label: { text: "What is your school’s local authority?", tag: "h1", size: "l" },
+              label: { text: "What's your school’s local authority?", tag: "h1", size: "l" },
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/choose_location.html.erb:14]:
```suggestion
              label: { text: "What’s your school’s local authority?", tag: "h1", size: "l" },
```

File: app/views/nominations/request_nomination_invite/choose_school.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<% content_for :title, "Find school" %>
+<% content_for :title, "What's the name of your school?" %>
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/choose_school.html.erb:1]:
```suggestion
<% content_for :title, "What’s the name of your school?" %>
```

File: app/views/nominations/request_nomination_invite/resend_email.html.erb [hunk 5]
```diff
@@ -4,27 +4,12 @@
...
-      <li>nominate an 'induction tutor' to set up your induction programme for early career teachers</li>
-    </ul>
     
-    <p>An induction tutor is someone who can:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>confirm details about your induction programme</li>
-      <li>add information about your early career teachers and mentors</li>
-    </ul>
+    <p>We need some information to help us find your school in the Get Information about Schools (GIAS) register.</p>        	
+    <p>We'll then send a link to the email address held in the register.</p>
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/resend_email.html.erb:9]:
```suggestion
    <p>We’ll then send a link to the email address held in the register.</p>
```

File: app/views/nominations/request_nomination_invite/resend_email.html.erb [hunk 6]
```diff
@@ -4,27 +4,12 @@
...
+    
+    <p>You can <a href="https://services.signin.education.gov.uk/">sign in to the DfE</a> to check that your school's email address is up to date before you start.</p>
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/resend_email.html.erb:11]:
```suggestion
    <p>You can <a href="https://services.signin.education.gov.uk/">sign in to the DfE</a> to check that your school’s email address is up to date before you start.</p>
```

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 7]
```diff
@@ -5,13 +5,15 @@
...
-gister, </li>
-      <li>
-       it will be an administration email, for example, office@, admin@ or head@
-      </li>
-       <li>If you don't receive an email, <%= govuk_link_to "sign in to DfE", "https://services.signin.education.gov.uk/" %> and make sure that your GIAS email address is up to date. When this has been updated you can <%= govuk_link_to "send a new link", choose_location_request_nomination_invite_path %></li>
-    </ul>
+       <p>Use the link to sign in to our service to:</p> 
+       <ul class="govuk-list govuk-list--bullet">
+        		<li>nominate an induction tutor for your school</li>
+        		<li>tell us how you'll run your early career teacher training programme</li>
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/success.html.erb:11]:
```suggestion
        		<li>tell us how you’ll run your early career teacher training programme</li>
```

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 8]
```diff
@@ -5,13 +5,15 @@
...
+       </ul>
+    
+       <h2 class="govuk-heading-m">If you do not get an email</h2>
+       <p><%= govuk_link_to "Sign in to the DfE", "https://services.signin.education.gov.uk/" %> and make sure that your school's email address is up to date in GIAS.
```
Comment (tonyheadford) [inline app/views/nominations/request_nomination_invite/success.html.erb:15]:
```suggestion
       <p><%= govuk_link_to "Sign in to the DfE", "https://services.signin.education.gov.uk/" %> and make sure that your school’s email address is up to date in GIAS. 
```

File: app/views/start/index.html.erb [hunk 9]
```diff
@@ -2,28 +2,24 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">Manage training for early career teachers</h1>
 
-    <p>Use this service to set up a 2 year training programme for your early career teachers (ECTs). You will need to
-      tell us:</p>
+    <p>Use this service to set up a 2-year training programme for your early career teachers (ECTs).</p> 
+    
+    <div class="govuk-inset-text">
+     <%= govuk_link_to "Read our step by step guidance (opens in new tab)", step_by_step_path, target: "_blank" %> to find out what's changed this year, and what you need to do.
```
Comment (tonyheadford) [inline app/views/start/index.html.erb:8]:
```suggestion
     <%= govuk_link_to "Read our step by step guidance (opens in new tab)", step_by_step_path, target: "_blank" %> to find out what’s changed this year, and what you need to do.
```

File: app/views/start/index.html.erb [hunk 10]
```diff
@@ -2,28 +2,24 @@
...
+    </div>
+    
+    <p>Sign in to this service to tell us:</p>
     <ul class="govuk-list govuk-list--bullet">
       <li>how you want to run your training</li>
-      <li>which accredited materials or training provider you will use</li>
+      <li>which accredited materials or training provider you'll use</li>
```
Comment (tonyheadford) [inline app/views/start/index.html.erb:14]:
```suggestion
      <li>which accredited materials or training provider you’ll use</li>
```

File: app/views/users/sessions/login_email_sent.html.erb [hunk 11]
```diff
@@ -2,15 +2,14 @@
...
-    <p class="govuk-body">This sign in link that will only work for this service. We do not use DfE sign in.</p>
-    
-    
-    <h2 class="govuk-heading-m">If you don't get an email to sign in</h2>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>you must be nominated or sent sign in details to access this service. <%= govuk_link_to "Find out how to get an account", check_account_path %></li>
-      <li>check your spam or junk folder</li>
-      <li>if you still need assistance, contact us: <%= render MailToSupportComponent.new %></li>
-    </ul>
+    <p class="govuk-body">If you have an account, we've sent you a link to sign in.</p>
```
Comment (tonyheadford) [inline app/views/users/sessions/login_email_sent.html.erb:5]:
```suggestion
    <p class="govuk-body">If you have an account, we’ve sent you a link to sign in.</p>
```

File: app/views/users/sessions/login_email_sent.html.erb [hunk 12]
```diff
@@ -2,15 +2,14 @@
...
+   
+    <h2 class="govuk-heading-m">If you do not get an email</h2>
+      		<ol class="govuk-list govuk-list--number">
+            <li>Check your spam or junk folder.</li>
+            <li>Try signing in again, making sure your email address is typed correctly and does not include any spaces.</li>
+		      </ol>
+            <p>If you're not registered, <%= govuk_link_to "find out how to get an account", check_account_path %>.</p>
```
Comment (tonyheadford) [inline app/views/users/sessions/login_email_sent.html.erb:12]:
```suggestion
            <p>If you’re not registered, <%= govuk_link_to "find out how to get an account", check_account_path %>.</p>
```

File: app/views/users/sessions/new.html.erb [hunk 13]
```diff
@@ -13,10 +13,9 @@
...
     <% end %>
-    <h2 class="govuk-heading-m">How you can access this service</h2>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>you must be nominated or sent sign in details to access this service. <%= govuk_link_to "Find out how to get an account", check_account_path %></li>
-      <li>we don’t use DfE sign in or passwords, you just need your account’s email address</li>
-    </ul>
+    <h2 class="govuk-heading-m">How to access this service</h2>
+      		<p>Enter your registered email address (the one we send emails to).</p>
+      		<p>Do not use your DfE Sign-in details.</p>
+        	<p>If you're not registered, <a class="govuk-link" href="how-to-get-an-account">find out how to get an account</a>.</p>
```
Comment (tonyheadford) [inline app/views/users/sessions/new.html.erb:19]:
```suggestion
        	<p>If you’re not registered, <a class="govuk-link" href="how-to-get-an-account">find out how to get an account</a>.</p>
```

File: app/views/users/sessions/redirect_from_magic_link.html.erb [hunk 14]
```diff
@@ -1,7 +1,7 @@
 <% content_for :title, "Complete sign in" %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l">You are now signed in</h1>
+    <h1 class="govuk-heading-l">You're now signed in</h1>
```
Comment (tonyheadford) [inline app/views/users/sessions/redirect_from_magic_link.html.erb:4]:
```suggestion
    <h1 class="govuk-heading-l">You’re now signed in</h1>
```

File: app/views/users/sessions/signed_out.html.erb [hunk 15]
```diff
@@ -1,2 +1,2 @@
-<h1 class="govuk-heading-xl">You are now signed out</h1>
+<h1 class="govuk-heading-xl">You're now signed out</h1>
```
Comment (tonyheadford) [inline app/views/users/sessions/signed_out.html.erb:1]:
```suggestion
<h1 class="govuk-heading-xl">You’re now signed out</h1>
```

**Review by tonyheadford — COMMENTED** [review]
Lots of wrong apostrophes used. If you are on a Mac @Lloyd-K, the smart apostrophe is `shift`+`option`+`]`

File: app/views/users/sessions/new.html.erb [hunk 16]
```diff
@@ -13,10 +13,9 @@
...
     <% end %>
-    <h2 class="govuk-heading-m">How you can access this service</h2>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>you must be nominated or sent sign in details to access this service. <%= govuk_link_to "Find out how to get an account", check_account_path %></li>
-      <li>we don’t use DfE sign in or passwords, you just need your account’s email address</li>
-    </ul>
+    <h2 class="govuk-heading-m">How to access this service</h2>
+      		<p>Enter your registered email address (the one we send emails to).</p>
+      		<p>Do not use your DfE Sign-in details.</p>
+        	<p>If you’re not registered, <a class="govuk-link" href="how-to-get-an-account">find out how to get an account</a>.</p>
```
Comment (tonyheadford) [inline app/views/users/sessions/new.html.erb:19]:
```suggestion
        	<p>If you’re not registered, <%= govuk_link_to "find out how to get an account", check_account_path %>.</p>
```

**Review by tonyheadford — APPROVED** [review]

File: app/views/nominations/request_nomination_invite/resend_email.html.erb [hunk 17]
```diff
@@ -4,27 +4,12 @@
...
     
-    <p>An induction tutor is someone who can:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>confirm details about your induction programme</li>
-      <li>add information about your early career teachers and mentors</li>
-    </ul>
+    <p>We need some information to help us find your school in the Get Information about Schools (GIAS) register.</p>        	
+    <p>We’ll then send a link to the email address held in the register.</p>
+    
+    <p>You can <a href="https://services.signin.education.gov.uk/">sign in to the DfE</a> to check that your school’s email address is up to date before you start.</p>
```
Comment (BroiSatse) [inline app/views/nominations/request_nomination_invite/resend_email.html.erb:11]: In theory all `p` tags should have `govuk-body` class, but it is really minor...

**Review by BroiSatse — APPROVED** [review]

**Pull request merged**